src/Particular.ThroughputCollector/IThroughputCollector.cs

```diff
@@ -7,7 +7,7 @@ public interface IThroughputCollector
         Task<List<EndpointThroughputSummary>> GetThroughputSummary();
         Task UpdateUserIndicatorsOnEndpoints(List<EndpointThroughputSummary> endpointsThroughputSummary);
         Task<BrokerSettings> GetBrokerSettingsInformation();
-        Task<ConnectionTestResults> TestConnectionSettings();
+        Task<ConnectionTestResults> TestConnectionSettings(CancellationToken cancellationToken = default);
```

Comment:
Well, all these need cancellation tokens, I raised it in https://github.com/orgs/Particular/projects/197/views/1?pane=issue&itemId=57288709